cli/daemon/run/testdata/echo_client/client.ts

```diff
@@ -516,7 +516,13 @@ function encodeQuery(parts: Record<string, string | string[]>): string {
     return pairs.join("&")
 }
 // CallParameters is the type of the parameters to a method call, but require headers to be a Record type
-type CallParameters = Omit<RequestInit, "method" | "body"> & { headers?: Record<string, string> }
+type CallParameters = Omit<RequestInit, "method" | "body"> & {
+    /** Any headers to be sent with the request */
+    headers?: Record<string, string>;
+
+    /** Any query parameters to be sent with the request */
+    queryFields?: Record<string, string | string[]>
```

Comment:
queryFields is quite verbose. Go calls it query, and it aligns with our struct tag.

Reply:
> Go calls it query,

However `RawQuery` (which is only 3 characters shorter 😉) in the URL object is the string of the entire query string, whereas I wanted to be clear that this was not a query string, but rather the fields which make up the query string.

Verboseness is good when it lowers the cognitive overhead needed to understand code at a glance.

> and it aligns with our struct tag.

That's fair, I'm still somewhat in favour of the verboseness to remove confusion maybe for frontend engineers who don't work with Encore directly.

What do you think?

Reply:
I think given that the type is a `Record` and we should document it, I think `query` is clear enough. If I had suggested `qry` I would be inclined to agree that it’s too terse :)